csrankings-b.csv

```diff
@@ -935,6 +935,7 @@ Byron J. Williams,Mississippi State University,http://www.fsnhp.msstate.edu/docs
 Byung Guk Kim,University of Massachusetts Lowell,https://www.uml.edu/Sciences/computer-science/faculty/kim-byung-guk.aspx,FkDwvF0AAAAJ
 Byung Ro Moon,Seoul National University,http://soar.snu.ac.kr/members,N_9XjY4AAAAJ
 Byung Seub Lee,University of Vermont,http://www.uvm.edu/~cems/?Page=employee/profile.php&SM=employee/_employeemenu.html&EmID=195,NOSCHOLARPAGE
+Byung-Cheol Min,Purdue University,https://web.ics.purdue.edu/~minb,eiAeSiAAAAAJ

```

Comment:
Please explain relationship to CS department. Only CS faculty (and those with courtesy appointments in CS) are eligible for inclusion.